config/prod/CMC-large-s3.yaml

```diff
@@ -0,0 +1,39 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template_path: remote/s3-bucket.yaml
+stack_name: cmc-large
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "cmc-wgs"
+  # The resource owner
+  OwnerEmail: 'kelsey.montgomery@sagebase.org'
+
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  SynapseUserName: 'kelsey'
+  # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:iam::945062467230:user/kelsey.montgomery@sagebase.org'
```

Comment:
is this your personal AWS account because that's not a Sage AWS account ID.  i believe this is OK just as long as that ARN does indeed exists.

Reply:
I updated the ARN to the Sage AWS account ID.